README.md

```diff
@@ -21,13 +21,34 @@ If you are already part of the DACCS Network and details about your node have ch
 In which case, please submit the pull request with the new URL and the Executive Committee will approve it quickly.
 
 When making changes to the node_registry.json file please only change or add the following values:
-  - url (the publicly accessible URL of your node)
-  - date_added (the date the node was originally added to the network, this should only be updated once)
-  - affiliation (the name of your organization, optional)
-  - description (a short description of your node, optional)
-  - icon_url (a publicly accessible url to an icon image for your node, optional)
-  - location (latitude and longitude of your organization, optional)
-  - contact (an email address that can be used by users to contact you if they have questions about your node)
+  - `date_added` (the date the node was originally added to the network, this should only be updated once)
+  - `affiliation` (the name of your organization, optional)
+  - `description` (a short description of your node, optional)
+  - `location` (latitude and longitude of your organization, optional)
+  - `contact` (an email address that can be used by users to contact you if they have questions about your node)
+  - `links -> version` (see links table below)
+  - `links -> collection` (see links table below)
+  - `links -> service` (see links table below)
+
+The `links` key should contain links that describe and provide access to your node. Please see the table below
+for a description of some link values that may be useful. Only `version`, `collection`, and `home` are required.
+
+| `rel`                                              | Meaning                                                                                                                                                         | 
```

Comment:
It is more readable to have a dedicated required/optional column than the inline notes.